code/datums/looping_sounds/_looping_sound.dm

```diff
@@ -37,16 +37,16 @@
 	var/skip_starting_sounds = FALSE
 	var/loop_started = FALSE
 	///Whether this looping sound takes character's sound accessibility preferences into account
-	var/player_sensitive = FALSE
+	var/loud_machine = FALSE
```

Comment:
Again, this should NOT be a per-sound thing. It should be clamped on volume itself.

Reply:
that wound then affect all machinery, and would make this option less of an accessibility issue for those who have problems dealing with constant loud noise, and just another volume button.

Reply:
The point is that there's no realistic way for us to maintain what counts as an inaccessible loud sound and what doesn't, so we have to clamp all volume. Not LOWER all volume, CLAMP the volume.